module/VuFind/src/VuFind/RecordTab/VersionsFactory.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
I think you can eliminate this class entirely if you instead use: https://github.com/vufind-org/vufind/blob/dc4677e100ecd786e5bb4b5b434a6ea08025aa9c/module/VuFind/src/VuFind/Service/ServiceWithConfigIniFactory.php (an existing generic factory for injecting config.ini).